.lgtm.yml

```diff
@@ -0,0 +1,7 @@
+extraction:
+    cpp:
+        after_prepare:
+            - export GNU_MAKE=make
+        index:
+            build_command:
+                - $GNU_MAKE -j2 -s
```

Comment:
Please use an indentation of 2 spaces when re-pushing.
(at least that's why I assume the convention here is)